runatlantis.io/docs/post-workflow-hooks.md

```diff
@@ -74,4 +76,5 @@ command](custom-workflows.html#custom-run-command).
   * `USER_NAME` - Username of the VCS user running command, ex. `acme-user`. During an autoplan, the user will be the Atlantis API user, ex. `atlantis`.
   * `COMMENT_ARGS` - Any additional flags passed in the comment on the pull request. Flags are separated by commas and
     every character is escaped, ex. `atlantis plan -- arg1 arg2` will result in `COMMENT_ARGS=\a\r\g\1,\a\r\g\2`.
+  * `OUTPUT_FILE` - An output file you can use to customize the success or failure status. ex. `echo 'failure' > $OUTPUT_FILE`.
```

Comment:
Is it better to use `OUTPUT_STATUS_FILE`? Cause `OUTPUT_FILE` means too large and atlantis may out put more contents in the future like https://github.com/runatlantis/atlantis/issues/936.